sympy/physics/wigner.py

```diff
@@ -69,7 +69,7 @@
 from sympy.matrices.dense import zeros
 from sympy.matrices.immutable import ImmutableMatrix
 from sympy.utilities.misc import as_int
-
+from fractions import Fraction
```

Comment:
We don't want to use Fraction. SymPy has its own versions of these things.

Reply:
My bad, I didn't think about using its own module for fractions,  now I have corrected it .  Let me know what do you think about this changes !